opal/util/ethtool.h

```diff
@@ -0,0 +1,25 @@
+#ifndef OPAL_ETHTOOL_H
+#define OPAL_ETHTOOL_H
+
+#include "opal_config.h"
+
+#if !defined(HAVE_DECL_ETHTOOL_CMD_SPEED)
+#if !defined(HAVE_STRUCT_ETHTOOL_CMD)
+struct ethtool_cmd;
+static inline unsigned int
+ethtool_cmd_speed(const struct ethtool_cmd *ep)
+{
+    return 0;
+}
+#else
+static inline unsigned int
+ethtool_cmd_speed(const struct ethtool_cmd *ep)
+{
+    return (ep->speed_hi << 16) | ep->speed;
+}
+#endif
+#endif
+
+unsigned int opal_ethtool_get_speed(const char *if_name);
```

Comment:
Please add at least a 1-line description of this function in a comment above the function declaration.